_includes/settlement-reconciliation.md

```diff
@@ -49,15 +49,15 @@ balance report summary is equivalent to the disbursement on the bank statement
 ##### Sales specification
 
 Provides a specification over sales for the given period. The sales total is
-specified per payment area (`CreditCard, Direct Debit, Factoring`) and
+specified per payment area (`CreditCard, Factoring`) and
 underlying payment instruments. Each sales row specify Quantity, Sum sales and
 Amount to pay out, the last one is only eligble **if Swedbank Pay handles the
 Settlement process**.
 
 ##### Fees specification
 
 Provides a specification over fees for the given period. The fees total is
```

Comment:
This paragraph seems like a duplication of the above "Sales specification" section. Not something that needs to be tackled in this PR, but could we do something about that?

Reply:
Noted!